README.md

```diff
@@ -116,7 +111,10 @@ opam install . --deps-only
 
 ### Building the undecidability library
 
-- `make all` builds the library
+#### Relevant `make` commands
+
+- `make TRAKHTENBROT/summary.vo` builds the review file `theories/TRAKHTENBROT/summary.v` for the Trakhtenbrot development (expect 2 minutes)
```

Comment:
do we want to formulate the instructions so explicit for reviewers?

Reply:
@dominik-kirst could you be more specific about what you find to much detailed ?

Reply:
Calling the summary `review` file and using the heading `Reviewing the Coq code` makes the readme sound like it is intended for reviewers only. If we use `Browsing the Coq code` or similar, the readme also works for external readers interested in the code. Or do you prefer explicitly addressing the reviewers only with the readme?

Reply:
Ok I see. No it is no intended for reviewers only so I will switch from _review_ to _browse_ or _read_ the Coq code.